deployments/arbitrum-goerli/usdc/migrations/1689112067_configurate_and_ens.ts

```diff
@@ -0,0 +1,263 @@
+
+export default migration('1689112067_configurate_and_ens', {
+  prepare: async (deploymentManager: DeploymentManager) => {
+    return {};
+  },
+
+  enact: async (deploymentManager: DeploymentManager, govDeploymentManager: DeploymentManager) => {
+    const trace = deploymentManager.tracer();
+    const ethers = deploymentManager.hre.ethers;
+    const { utils } = ethers;
+
+    const cometFactory = await deploymentManager.fromDep('cometFactory', 'arbitrum-goerli', 'usdc.e');
+    const usdceComet = await deploymentManager.fromDep('usdceComet', 'arbitrum-goerli', 'usdc.e', 'comet');
+    const {
+      bridgeReceiver,
+      timelock: l2Timelock,
+      comet,
+      cometAdmin,
+      configurator,
+      rewards,
+    } = await deploymentManager.getContracts();
+
+    const {
+      arbitrumInbox,
+      arbitrumL1GatewayRouter,
+      timelock,
+      governor,
+      USDC,
+      COMP,
+      CCTPTokenMessenger,
+    } = await govDeploymentManager.getContracts();
+
+    // CCTP destination domain for Arbitrum
+    const ArbitrumDestinationDomain = 3;
+    const USDCAmountToBridge = exp(10, 6);
+    const refundAddress = l2Timelock.address;
+    const configuration = await getConfigurationStruct(deploymentManager);
+    const setFactoryCalldata = await calldata(
+      configurator.populateTransaction.setFactory(comet.address, cometFactory.address)
+    );
+    
+    const setConfigurationCalldata = await calldata(
+      configurator.populateTransaction.setConfiguration(comet.address, configuration)
+    );
+
+    const deployAndUpgradeToCalldata = utils.defaultAbiCoder.encode(
+      ['address', 'address'],
+      [configurator.address, comet.address]
+    );
+
+    const setRewardConfigCalldata = utils.defaultAbiCoder.encode(
+      ['address', 'address'],
+      [comet.address, arbitrumCOMPAddress]
+    );
+
+    const turnOffUSDCeCometSupplySpeedCalldata = utils.defaultAbiCoder.encode(
+      ['address', 'uint64'],
+      [usdceComet.address, 0]
+    );
+
+    const turnOffUSDCeCometBorrowSpeedCalldata = utils.defaultAbiCoder.encode(
+      ['address', 'uint64'],
+      [usdceComet.address, 0]
+    );
+
+    const deployAndUpgradeToUSDCeCometCalldata = utils.defaultAbiCoder.encode(
+      ['address', 'address'],
+      [configurator.address, usdceComet.address]
+    );
+
+    const l2ProposalData = utils.defaultAbiCoder.encode(
+      ['address[]', 'uint256[]', 'string[]', 'bytes[]'],
+      [
+        [configurator.address, configurator.address, cometAdmin.address, rewards.address, configurator.address, configurator.address, cometAdmin.address],
+        [0, 0, 0, 0, 0, 0, 0],
+        [
+          'setFactory(address,address)',
+          'setConfiguration(address,(address,address,address,address,address,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint104,uint104,uint104,(address,address,uint8,uint64,uint64,uint64,uint128)[]))',
```

Comment:
Instead of `setConfiguration`, you can also call `setBaseTrackingSupplySpeed` and `setBaseTrackingBorrowSpeed` if those are the only configs you are modifying.